src/algorithms.rs

```diff
@@ -1,4 +1,5 @@
 use std::str::FromStr;
+use self::Algorithm::*;
```

Comment:
To each their own. In limited use (Enumerations which are the primary focus of a module, custom preludes, and `use super::*` in test modules), I find glob imports pretty nice. But I'll remove their use.